lib/matplotlib/widgets.py

```diff
@@ -380,11 +382,24 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
             The color of the line at the *valinit* position. Set to ``'none'``
             for no line.
 
+        track_color : color, default: 'lightgrey'
+            The color of the background track. The track is accessible for
+            further styling via the *track* attribute.
+
+        handle_facecolor : color, default: 'white'
+            The facecolor of the circular slider handle.
+
+        handle_edgecolor : color, default: '.75'
+            The edgecolor of the circle slider handle.
+
+        handle_size : int, default: 10
+            The size of the circular slider handle in points.
+
         Notes
         -----
         Additional kwargs are passed on to ``self.poly`` which is the
-        `~matplotlib.patches.Rectangle` that draws the slider knob.  See the
-        `.Rectangle` documentation for valid property names (``facecolor``,
+        `~matplotlib.patches.Polygon` that draws the slider knob.  See the
```

Comment:
https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.axvspan.html

`ax{h,v}span` returns a `Polygon` not a `Rectangle` patch.